### DISCUSSION TITLE
The models don't work once downloaded!

### DISCUSSION THREAD

**Discussion by Bonjonbon** [TheLastBen/fast-stable-diffusion#2659] [Q&A] [answered]
I had just got done training a new model and downloaded it to use in A1111. But when I tried to switch to the newly trained model, I kept getting an error. I've tried resetting A1111 but it didn't work, I started over and made a new model with the same images, but that didn't work either. I deleted A1111 and reinstalled a fresh one but the model still doesn't work!

But oddly it works on your colab version of A111 just fine but not when I download it. This only started recently, so maybe there was an update that breaks models after finishing them. If you need to see the error here you go:

changing setting sd_model_checkpoint to Test33.ckpt: AttributeError
Traceback (most recent call last):
  File "D:\A1111\stable-diffusion-webui\modules\options.py", line 140, in set
    option.onchange()
  File "D:\A1111\stable-diffusion-webui\modules\call_queue.py", line 13, in f
    res = func(*args, **kwargs)
  File "D:\A1111\stable-diffusion-webui\modules\initialize_util.py", line 170, in <lambda>
    shared.opts.onchange("sd_model_checkpoint", wrap_queued_call(lambda: sd_models.reload_model_weights()), call=False)
  File "D:\A1111\stable-diffusion-webui\modules\sd_models.py", line 741, in reload_model_weights
    state_dict = get_checkpoint_state_dict(checkpoint_info, timer)
  File "D:\A1111\stable-diffusion-webui\modules\sd_models.py", line 315, in get_checkpoint_state_dict
    res = read_state_dict(checkpoint_info.filename)
  File "D:\A1111\stable-diffusion-webui\modules\sd_models.py", line 301, in read_state_dict
    sd = get_state_dict_from_checkpoint(pl_sd)
  File "D:\A1111\stable-diffusion-webui\modules\sd_models.py", line 245, in get_state_dict_from_checkpoint
    pl_sd = pl_sd.pop("state_dict", pl_sd)
AttributeError: 'NoneType' object has no attribute 'pop'

**Comment by grumblebumble** [comment]
I am having the exact same problem, same error and eveything on automatic1111. Using Stable Diffusion 1.5 for training.

**Reply by Bonjonbon** [reply]
Yeah, I'm training on a 1.5 model too, I forgot to mention that.

**Reply by grumblebumble** [reply, marked as answer]
I found what appears to be a fix for now. Edit webui-user.bat and add this:

set COMMANDLINE_ARGS=--disable-safe-unpickle

the "--disable-safe-unpickle" is the key.

**Reply by Bonjonbon** [reply]
It works! Thank you! But I'm not sure this is best solution for it as this mean any model I download and use, is not being check for safety. But this works for my trained models.

**Comment by TheLastBen** [comment]
is your A1111 up to date ?

**Reply by grumblebumble** [reply]
Yes, I'm running the latest Automatic1111 commit.

**Reply by Bonjonbon** [reply]
Yep, I even reinstall it to make sure I was. I'm not sure what's going on. Do you have an alternate solution to this?
